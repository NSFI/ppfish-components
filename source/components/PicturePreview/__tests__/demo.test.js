import demoTest from '../../../tests/demoTest';

demoTest('PicturePreview');
