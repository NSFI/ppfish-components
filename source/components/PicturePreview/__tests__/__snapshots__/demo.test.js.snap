// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./source/components/PicturePreview/demo/App.js correctly 1`] = `
initialize {
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
